lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +91,50 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config = json.load(file)
+        return ConfigDTO(
+            seed_urls=config["seed_urls"],
+            total_articles_to_find_and_parse=config["total_articles_to_find_and_parse"],
+            headers=config["headers"],
+            encoding=config["encoding"],
+            timeout=config["timeout"],
+            should_verify_certificate=config["should_verify_certificate"],
+            headless_mode=config["headless_mode"]
+        )
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
```

Comment:
we need to operate with DTO. why do we need it then if we ignore its existence and always refer to raw files as source of truth?

Reply:
I initially used DTO here, but documentation told me to specifically call the validation **before** extracting config content to DTO (line 279 in README.rst) 🥲

Reply:
why do not you create DTO inside this function then? is it so expensive?

Reply:
Ok, added it